content/articles/2019-03-ggplot2-3-2-0.Rmarkdown

```diff
@@ -0,0 +1,116 @@
+---
+title: 'ggplot2 3.2.0'
+author: Thomas Lin Pedersen
+date: '2019-05-20'
+slug: ggplot2-3-2-0
+description: > 
+  ggplot2 3.2.0 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/eH_ftJYhaTY
+  author: chuttersnap
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're thrilled to announce the release of [ggplot2 3.2.0](https://ggplot2.tidyverse.org) on CRAN. ggplot2 is a system for declaratively creating graphics, based on The Grammar of Graphics. You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
+
+The 3.2.0 release is a minor release with focus on performance improvements, but also includes a few new features and a range of bug fixes. There are no breaking changes in this release, though a few changes may affect the visual appearance of plots in minor ways, noted below. Further, there are a few changes that developers of extension packages may need to take into account, but won't affect regular users. For a full overview of the changes, please see the [release notes](https://ggplot2.tidyverse.org/news/index.html).
+
+This release also includes a range of contributions made during our tidyverse developer day in Austin, many from first-time contributors. We hope they've become inspired to continue taking part in the development.
+
+Lastly, this release also sees the entrance of Hiroaki Yutani (*yutannihilation* on both [Github](https://github.com/yutannihilation) and [Twitter](https://twitter.com/yutannihilat_en)) into the core developer team. Hiroaki has been amazing in tackling small and large issues both before and after he joined the team, and we are very lucky to have him.
```

Comment:
I would suggest "large and small" instead of "small and large". If you lead with "small", you give the impression most of the contributions were not that important.